README.md
# CytoExploreRData
Flow cytometry data for use in CytoExploreR.


NEWS.md
DataVersion: 0.1.81
=======================


DataVersion: 0.1.80
=======================
Add Hoechst compensation control to Compensation dataset.

DataVersion: 0.1.79
=======================
Convert datasets to flowSets after barcoding

DataVersion: 0.1.78
=======================
Add event barcodes to Activation and Compensation datasets

DataVersion: 0.1.77
=======================
Tidy imports.

DataVersion: 0.1.76
=======================
Bump package version.

DataVersion: 0.1.75
=======================
* Changed: Activation
* Changed: Compensation
Add Compensation and Activation GatingSets to extdata.

DataVersion: 0.1.74
=======================
Add marker assignments to Compensation.

DataVersion: 0.1.73
=======================
* Changed: Activation
* Changed: Compensation
Update gatingTemplates to avoid dummy gates when gating empty populations.

DataVersion: 0.1.72
=======================
* Changed: Activation
* Changed: Compensation
Remove excess files.

DataVersion: 0.1.71
=======================
* Changed: Compensation
Update Activation datasets.

DataVersion: 0.1.70
=======================
* Added: Activation
* Changed: Compensation
test

DataVersion: 0.1.69
=======================
* Deleted: Activation
* Changed: Compensation
test

DataVersion: 0.1.68
=======================
* Changed: Activation
* Changed: Compensation
test

DataVersion: 0.1.67
=======================
* Changed: Activation
* Changed: Compensation
test

DataVersion: 0.1.66
=======================
* Changed: Activation
* Changed: Compensation
test

DataVersion: 0.1.65
=======================
* Changed: Activation
* Changed: Compensation
test

DataVersion: 0.1.64
=======================
* Changed: Activation
* Changed: Compensation
Comment out pData and GatingSet lines in Activation.

DataVersion: 0.1.63
=======================
* Changed: Activation
* Changed: Compensation
Remove pData from Activation.

DataVersion: 0.1.62
=======================
* Changed: Activation
* Changed: Compensation
Remove pData assignments.

DataVersion: 0.1.61
=======================
* Changed: Compensation
Update cytoset.

DataVersion: 0.1.60
=======================
* Changed: Activation
* Changed: Compensation
Version 1.0.1

DataVersion: 0.1.59
=======================
* Changed: Activation
* Changed: Compensation
Convert Activation to GatingSet.

DataVersion: 0.1.58
=======================
* Changed: Activation
* Changed: Compensation
Activation updates.

DataVersion: 0.1.57
=======================
Activation GatingSet.

DataVersion: 0.1.56
=======================
* Changed: Activation
* Changed: Compensation
Remove old CytoRSuiteData references and update to new Activation gatingTemplate.

DataVersion: 0.1.55
=======================
Update spill slots of Activation and Compensation gatingTemplate.

DataVersion: 0.1.54
=======================
* Changed: Activation
* Changed: Compensation


DataVersion: 0.1.53
=======================
* Changed: Compensation
Keep Activation as flowSet to prevent ncdf file accessibility issues.

DataVersion: 0.1.52
=======================
* Changed: Compensation
Update Activation files to remove old filenames.

DataVersion: 0.1.51
=======================
* Changed: Compensation
Automatically correct file order using gtools mixedsort.

DataVersion: 0.1.50
=======================
CytoExploreR name changes.

DataVersion: 0.1.49
=======================
* Changed: Activation
* Changed: Compensation
Update Compensation gatingTemplate.

DataVersion: 0.1.48
=======================
* Changed: Activation
* Changed: Compensation
Update Activation gatingTemplate.

DataVersion: 0.1.47
=======================
* Changed: Compensation
Correct experiment details in Activation.

DataVersion: 0.1.46
=======================
* Changed: Activation
* Changed: Compensation


DataVersion: 0.1.45
=======================
* Changed: Compensation
Add unstained control to Activation and update gatingTemplates.

DataVersion: 0.1.44
=======================
* Changed: Compensation
Fix order in Activation and add experiment details.

DataVersion: 0.1.43
=======================
* Changed: Compensation
Activation file name changes.

DataVersion: 0.1.42
=======================
* Changed: Activation
* Changed: Compensation


DataVersion: 0.1.41
=======================
* Changed: Activation
* Changed: Compensation


DataVersion: 0.1.40
=======================
* Changed: Activation
* Changed: Compensation
Update compensation dataset and gatingTemplate.

DataVersion: 0.1.39
=======================
* Changed: Activation
* Changed: Compensation


DataVersion: 0.1.38
=======================
* Changed: Activation
* Changed: Compensation
Updsate documentation for Activation dataset

DataVersion: 0.1.37
=======================
Update CytoRSuiteData to include more Activation files and experiment details.

DataVersion: 0.1.36
=======================
Name change to Compensation-Channels.csv

DataVersion: 0.1.35
=======================
* Changed: Activation
* Changed: Compensation


DataVersion: 0.1.34
=======================
* Changed: Activation
* Changed: Compensation
Update Activation gatingTemplate with new gating method and preprocessing names

DataVersion: 0.1.33
=======================
* Changed: Activation
* Changed: Compensation


DataVersion: 0.1.32
=======================
* Changed: Activation
* Changed: Compensation
Update documentation for Compensation gatingTemplate.

DataVersion: 0.1.31
=======================
* Added: Compensation_gatingTemplate
* Changed: Activation
* Changed: Compensation
Add Compensation gatingTemplate and channel match files for tests and examples.

DataVersion: 0.1.30
=======================
* Changed: Activation
* Changed: Compensation
Update gatingTemplate csv file to remove groupBy.

DataVersion: 0.1.29
=======================
* Changed: Activation
* Changed: Compensation
Change name Example-gatingTemplate.csv to Activation-gatingTemplate.csv

DataVersion: 0.1.28
=======================

R/CytoRSuiteData.R
#' CytoRSuiteData
#' A data package for CytoRSuiteData.
#' @docType package
#' @aliases CytoRSuiteData-package
#' @title Package Title
#' @name CytoRSuiteData
#' @description A description of the data package
#' @details Use \code{data(package='CytoRSuiteData')$results[, 3]} to see a list of available data sets in this data package
#'     and/or DataPackageR::load_all
#' _datasets() to load them.
#' @seealso
#' \link{Compensation_gatingTemplate}
NULL







#' CytoRSuite gatingTemplate used to gate the Activation data set.
#' @name Activation_gatingTemplate
#' @docType data
#' @title CytoRSuite Activation gatingTemplate.
#' @format a \code{gatingTemplate} object containing the following 9 gates:
#'   \itemize{ \item{\strong{Cells}} \item{\strong{Single Cells}}
#'   \item{\strong{Live Cells}} \item{\strong{T Cells}} \item{\strong{Dendritic
#'   Cells}} \item{\strong{CD4 T Cells}} \item{\strong{CD8 T Cells}}
#'   \item{\strong{CD69+ CD4 T Cells}} \item{\strong{CD69+ CD8 T Cells}} }
#' @source The Activation_gatingTemplate was generated using CytoRSuite to
#'   analyse Activation data set.
#' @seealso \link{CytoRSuiteData}
#' @seealso \link{Compensation}
#' @seealso \link{Activation}
#' @author Dillon Hammill (Dillon.Hammill@anu.edu.au)
NULL







#' @title CytoRSuite Compensation Data Set.
#' @description A collection of single stain compensation controls and an
#'   unstained control used to compensate the Activation data set.
#' @name Compensation
#' @docType data
#' @format A \code{flowSet} containing the 7 compensation controls.
#' @details Compensation controls are as follows: \itemize{
#'   \item{\strong{Compensation-7AAD.fcs}}
#'   \item{\strong{Compensation-AF700.fcs}} \item{\strong{Compensation-APC.fcs}}
#'   \item{\strong{Compensation-APC-Cy7.fcs}}
#'   \item{\strong{Compensation-FITC.fcs}} \item{\strong{Compensation-PE.fcs}}
#'   \item{\strong{Compensation-Unstained.fcs}}}
#' @source Compensation controls used for an in vitro OT-I/II T cell activation
#'   assay.
#' @seealso \link{CytoRSuiteData}
#' @seealso \link{Activation}
#' @seealso \link{Activation_gatingTemplate}
#' @author Dillon Hammill (Dillon.Hammill@anu.edu.au)
NULL





#' @title CytoRSuite Activation Data Set
#' @description Data from an in vitro OT-I/II T cell activation assay.
#' @name Activation
#' @docType data
#' @format a \code{flowSet} containing 33 samples with 4 OVA concentrations
#'   (0,0.005,0.05 and 0.5) and 4 distinct treatments
#'   ("Stim-A","Stim-B","Stim-C" and "Stim-D") in duplicate. The last sample is
#'   a universal unstained control which can be used to aid in setting
#'   appropriate gates.
#' @details \describe{ An experiment performed to assess the in vitro activation
#'   of OT-I and OT-II transgenic T cells co-cultured with OVA-presenting
#'   dendritic cells (DCs). This data set is comprised of 32 samples saved as a
#'   \code{flowSet} object. Samples include 4 OVA concentrations and 4 distinct
#'   treatment groups in duplicate.
#'
#'   Pooled DCs & T cells were labelled as follows: \itemize{
#'   \item{\strong{Hoechst 33258} a DNA intercalating dye which fluoresces in
#'   Alexa Fluor 405-A and Alexa Fluor 430-A channels.}
#'   \item{\strong{CD11c-APC-Cy7} to identify DCs within the samples.}
#'   \item{\strong{Va2-PE} to identify cells which express the OVA-specific TCR
#'   (i.e. OT-I and OT-II T cells).} \item{\strong{CD8-FITC} to identify the
#'   CD8+ OT-I transgenic T cells (Alexa Fluor 488-A).} \item{\strong{CD4-Alexa
#'   Fluor 700} to identify the CD4+ OT-II transgenic T cells (Alexa Fluor
#'   700-A).} \item{\strong{CD44-APC} to assess the extent of T cell activation
#'   (Alexa Fluor 647-A).} \item{\strong{CD69-PerCP Cy5.5} to assess the extent
#'   of T cell activation (7-AAD-A).} }}
#' @source The data comes from an in vitro OT-I/II T cell activation assay.
#' @seealso \link{CytoRSuiteData}
#' @seealso \link{Compensation}
#' @seealso \link{Activation_gatingTemplate}
#' @author Dillon Hammill (Dillon.Hammill@anu.edu.au)
NULL





#' @title CytoRSuite gatingTemplate used gate the Compensation data set.
#' @description A gatingTemplate used to gate the Compensation data set prior to
#'   spillover calculation.
#' @name Compensation_gatingTemplate
#' @docType data
#' @title Descriptive data title
#' @format a \code{gatingTemplate} containing the following 3 gates: \itemize{
#'   \item{\strong{nonDebris}} \item{\strong{Cells}} \ item{\strong{Single
#'   Cells}}}
#' @source The compensation gatingTemplate was generated using CytoRSuite to
#'   gate the compensation controls prior to claculation of fluorescent
#'   spillover.
#' @seealso \link{CytoRSuiteData}
#' @seealso \link{Compensation}
#' @author Dillon Hammill (Dillon.Hammill@anu.edu.au)
NULL


data-raw/documentation.R
#' CytoRSuiteData
#' A data package for CytoRSuiteData.
#' @docType package
#' @aliases CytoRSuiteData-package
#' @title Package Title
#' @name CytoRSuiteData
#' @description A description of the data package
#' @details Use \code{data(package='CytoRSuiteData')$results[, 3]} to see a list of available data sets in this data package
#'     and/or DataPackageR::load_all
#' _datasets() to load them.
#' @seealso
#' \link{Compensation_gatingTemplate}
NULL






#' CytoRSuite gatingTemplate used to gate the Activation data set.
#' @name Activation_gatingTemplate
#' @docType data
#' @title CytoRSuite Activation gatingTemplate.
#' @format a \code{gatingTemplate} object containing the following 9 gates:
#'   \itemize{ \item{\strong{Cells}} \item{\strong{Single Cells}}
#'   \item{\strong{Live Cells}} \item{\strong{T Cells}} \item{\strong{Dendritic
#'   Cells}} \item{\strong{CD4 T Cells}} \item{\strong{CD8 T Cells}}
#'   \item{\strong{CD69+ CD4 T Cells}} \item{\strong{CD69+ CD8 T Cells}} }
#' @source The Activation_gatingTemplate was generated using CytoRSuite to
#'   analyse Activation data set.
#' @seealso \link{CytoRSuiteData}
#' @seealso \link{Compensation}
#' @seealso \link{Activation}
#' @author Dillon Hammill (Dillon.Hammill@anu.edu.au)
NULL






#' @title CytoRSuite Compensation Data Set.
#' @description A collection of single stain compensation controls and an
#'   unstained control used to compensate the Activation data set.
#' @name Compensation
#' @docType data
#' @format A \code{flowSet} containing the 7 compensation controls.
#' @details Compensation controls are as follows: \itemize{
#'   \item{\strong{Compensation-7AAD.fcs}}
#'   \item{\strong{Compensation-AF700.fcs}} \item{\strong{Compensation-APC.fcs}}
#'   \item{\strong{Compensation-APC-Cy7.fcs}}
#'   \item{\strong{Compensation-FITC.fcs}} \item{\strong{Compensation-PE.fcs}}
#'   \item{\strong{Compensation-Unstained.fcs}}}
#' @source Compensation controls used for an in vitro OT-I/II T cell activation
#'   assay.
#' @seealso \link{CytoRSuiteData}
#' @seealso \link{Activation}
#' @seealso \link{Activation_gatingTemplate}
#' @author Dillon Hammill (Dillon.Hammill@anu.edu.au)
NULL




#' @title CytoRSuite Activation Data Set
#' @description Data from an in vitro OT-I/II T cell activation assay.
#' @name Activation
#' @docType data
#' @format a \code{flowSet} containing 32 samples with 4 OVA concentrations
#'   (0,0.005,0.05 and 0.5) and 4 distinct treatments
#'   ("Stim-A","Stim-B","Stim-C" and "Stim-D") in duplicate.
#' @details \describe{ An experiment performed to assess the in vitro activation
#'   of OT-I and OT-II transgenic T cells co-cultured with OVA-presenting
#'   dendritic cells (DCs). This data set is comprised of 32 samples saved as a
#'   \code{flowSet} object. Samples include 4 OVA concentrations and 4 distinct
#'   treatment groups in duplicate.
#'
#'   Pooled DCs & T cells were labelled as follows: \itemize{
#'   \item{\strong{Hoechst 33258} a DNA intercalating dye which fluoresces in
#'   Alexa Fluor 405-A and Alexa Fluor 430-A channels.}
#'   \item{\strong{CD11c-APC-Cy7} to identify DCs within the samples.}
#'   \item{\strong{Va2-PE} to identify cells which express the OVA-specific TCR
#'   (i.e. OT-I and OT-II T cells).} \item{\strong{CD8-FITC} to identify the
#'   CD8+ OT-I transgenic T cells (Alexa Fluor 488-A).} \item{\strong{CD4-Alexa
#'   Fluor 700} to identify the CD4+ OT-II transgenic T cells (Alexa Fluor
#'   700-A).} \item{\strong{CD44-APC} to assess the extent of T cell activation
#'   (Alexa Fluor 647-A).} \item{\strong{CD69-PerCP Cy5.5} to assess the extent
#'   of T cell activation (7-AAD-A).} }}
#' @source The data comes from an in vitro OT-I/II T cell activation assay.
#' @seealso \link{CytoRSuiteData}
#' @seealso \link{Compensation}
#' @seealso \link{Activation_gatingTemplate}
#' @author Dillon Hammill (Dillon.Hammill@anu.edu.au)
NULL




#' @title CytoRSuite gatingTemplate used gate the Compensation data set.
#' @description A gatingTemplate used to gate the Compensation data set prior to
#'   spillover calculation.
#' @name Compensation_gatingTemplate
#' @docType data
#' @title Descriptive data title
#' @format a \code{gatingTemplate} containing the following 3 gates: \itemize{
#'   \item{\strong{nonDebris}} \item{\strong{Cells}} \ item{\strong{Single
#'   Cells}}}
#' @source The compensation gatingTemplate was generated using CytoRSuite to
#'   gate the compensation controls prior to claculation of fluorescent
#'   spillover.
#' @seealso \link{CytoRSuiteData}
#' @seealso \link{Compensation}
#' @author Dillon Hammill (Dillon.Hammill@anu.edu.au)
NULL


R/CytoExploreRData.R
#' CytoExploreRData
#' A data package for CytoExploreRData.
#' @docType package
#' @aliases CytoExploreRData-package
#' @title Package Title
#' @name CytoExploreRData
#' @description A description of the data package
#' @details Use \code{data(package='CytoExploreRData')$results[, 3]} to see a list of available data sets in this data package
#'     and/or DataPackageR::load_all
#' _datasets() to load them.
#' @seealso
#' \link{Compensation_gatingTemplate}
NULL







#' CytoExploreR gatingTemplate used to gate the Activation data set.
#' @name Activation_gatingTemplate
#' @docType data
#' @title CytoExploreR Activation gatingTemplate.
#' @format a \code{gatingTemplate} object containing the following 9 gates:
#'   \itemize{ \item{\strong{Cells}} \item{\strong{Single Cells}}
#'   \item{\strong{Live Cells}} \item{\strong{T Cells}} \item{\strong{Dendritic
#'   Cells}} \item{\strong{CD4 T Cells}} \item{\strong{CD8 T Cells}}
#'   \item{\strong{CD69+ CD4 T Cells}} \item{\strong{CD69+ CD8 T Cells}} }
#' @source The Activation_gatingTemplate was generated using CytoExploreR to
#'   analyse Activation data set.
#' @seealso \link{CytoExploreRData}
#' @seealso \link{Compensation}
#' @seealso \link{Activation}
#' @author Dillon Hammill (Dillon.Hammill@anu.edu.au)
NULL







#' @title CytoExploreR Compensation Data Set.
#' @description A collection of single stain compensation controls and an
#'   unstained control used to compensate the Activation data set.
#' @name Compensation
#' @docType data
#' @format A \code{flowSet} containing the 7 compensation controls.
#' @details Compensation controls are as follows: \itemize{
#'   \item{\strong{Compensation-7AAD.fcs}}
#'   \item{\strong{Compensation-AF700.fcs}} \item{\strong{Compensation-APC.fcs}}
#'   \item{\strong{Compensation-APC-Cy7.fcs}}
#'   \item{\strong{Compensation-FITC.fcs}} \item{\strong{Compensation-PE.fcs}}
#'   \item{\strong{Compensation-Unstained.fcs}}}
#' @source Compensation controls used for an in vitro OT-I/II T cell activation
#'   assay.
#' @seealso \link{CytoExploreRData}
#' @seealso \link{Activation}
#' @seealso \link{Activation_gatingTemplate}
#' @author Dillon Hammill (Dillon.Hammill@anu.edu.au)
NULL





#' @title CytoExploreR Activation Data Set
#' @description Data from an in vitro OT-I/II T cell activation assay.
#' @name Activation
#' @docType data
#' @format a \code{flowSet} containing 33 samples with 4 OVA concentrations
#'   (0,0.005,0.05 and 0.5) and 4 distinct treatments
#'   ("Stim-A","Stim-B","Stim-C" and "Stim-D") in duplicate. The last sample is
#'   a universal unstained control which can be used to aid in setting
#'   appropriate gates.
#' @details \describe{ An experiment performed to assess the in vitro activation
#'   of OT-I and OT-II transgenic T cells co-cultured with OVA-presenting
#'   dendritic cells (DCs). This data set is comprised of 32 samples saved as a
#'   \code{flowSet} object. Samples include 4 OVA concentrations and 4 distinct
#'   treatment groups in duplicate.
#'
#'   Pooled DCs & T cells were labelled as follows: \itemize{
#'   \item{\strong{Hoechst 33258} a DNA intercalating dye which fluoresces in
#'   Alexa Fluor 405-A and Alexa Fluor 430-A channels.}
#'   \item{\strong{CD11c-APC-Cy7} to identify DCs within the samples.}
#'   \item{\strong{Va2-PE} to identify cells which express the OVA-specific TCR
#'   (i.e. OT-I and OT-II T cells).} \item{\strong{CD8-FITC} to identify the
#'   CD8+ OT-I transgenic T cells (Alexa Fluor 488-A).} \item{\strong{CD4-Alexa
#'   Fluor 700} to identify the CD4+ OT-II transgenic T cells (Alexa Fluor
#'   700-A).} \item{\strong{CD44-APC} to assess the extent of T cell activation
#'   (Alexa Fluor 647-A).} \item{\strong{CD69-PerCP Cy5.5} to assess the extent
#'   of T cell activation (7-AAD-A).} }}
#' @source The data comes from an in vitro OT-I/II T cell activation assay.
#' @seealso \link{CytoExploreRData}
#' @seealso \link{Compensation}
#' @seealso \link{Activation_gatingTemplate}
#' @author Dillon Hammill (Dillon.Hammill@anu.edu.au)
NULL





#' @title CytoExploreR gatingTemplate used gate the Compensation data set.
#' @description A gatingTemplate used to gate the Compensation data set prior to
#'   spillover calculation.
#' @name Compensation_gatingTemplate
#' @docType data
#' @title Descriptive data title
#' @format a \code{gatingTemplate} containing the following 3 gates: \itemize{
#'   \item{\strong{nonDebris}} \item{\strong{Cells}} \ item{\strong{Single
#'   Cells}}}
#' @source The compensation gatingTemplate was generated using CytoExploreR to
#'   gate the compensation controls prior to claculation of fluorescent
#'   spillover.
#' @seealso \link{CytoExploreRData}
#' @seealso \link{Compensation}
#' @author Dillon Hammill (Dillon.Hammill@anu.edu.au)
NULL
